### PR TITLE
Updates to brms overview page and bpm suite overview page.

### PR DESCRIPTION
**Pull request by eschabell**

### CODE REVIEW

**Comment by eschabell** [comment]
Same changes pushed to developer.redhat.com already.

**Comment by eschabell** [comment]
Preview is good, please merge to production.

**Comment by paulrobinson** [comment]
Please can you squash the commits.

**Comment by eschabell** [comment]
I think I squashed them.. but rebase here was an adventure....

**Comment by paulrobinson** [comment]
try doing a pull with rebase on upstream master. I think the command is:

git pull --rebase upstream master.

I have an alias for this, so forget what the basic command is.

**Comment by eschabell** [comment]
$ git pull --rebase upstream master
From github.com:redhat-developer/developers.redhat.com
 * branch            master     -> FETCH_HEAD
First, rewinding head to replay your work on top of it...
Applying: Adjustements to brms product overview page.
Applying: Adjustements to brms product overview page.
Using index info to reconstruct a base tree...
A	images/products/brms/capture.png
M	products/brms/overview.adoc
Falling back to patching base and 3-way merge...
No changes -- Patch already applied.

pulled and pushed, seems to have updated now?

**Comment by paulrobinson** [comment]
I'm still seeing a lot of commitsL: https://www.dropbox.com/s/kbfzadiy81b2x6h/Screenshot%202015-12-04%2018.14.46.png?dl=0

If you point out which commit you really want in, I can make a new PR with just that in, if you having trouble?

**Comment by eschabell** [comment]
Thanks, just the bottom one is fine... not sure why I can't get that done...

Regards,
Eric D. Schabell

File: products/brms/_common/product.yml [hunk 1]
```diff
@@ -113,7 +115,7 @@ downloads:
       release_notes:
       source:
         size: 154mb
-        filename: jboss-bpmsuite-6.0.1.GA-source.zip
+        filename: jboss-bpmsuite-6.0.1.GA-source.zip
```
Comment (LightGuard) [inline products/brms/_common/product.yml]: Please remove the extra whitespace

File: products/brms/_common/product.yml [hunk 2]
```diff
@@ -122,4 +124,3 @@ upstream_projects:
 featured_items:
   - image_url: http://design.jboss.org/jbossdeveloper/sitedesign/jbossdeveloper_1_0/makerfiles/images/developermaterials_brms.png
     url: /quickstarts/brms/helloworld-brms
-
```
Comment (LightGuard) [inline products/brms/_common/product.yml]: Why did you remove the trailing newline in the file?

File: products/bpmsuite/_common/product.yml [hunk 3]
```diff
@@ -106,4 +106,3 @@ upstream_projects:
 featured_items:
   - image_url: http://design.jboss.org/jbossdeveloper/sitedesign/jbossdeveloper_1_0/makerfiles/images/developermaterials_bpmsuite.png
     url: /quickstarts/brms/tasks-bpmsuite
-
```
Comment (LightGuard) [inline products/bpmsuite/_common/product.yml]: Why remove the trailing newline to the file?

File: products/brms/overview.adoc [hunk 4]
```diff
@@ -40,9 +44,10 @@ Create complete business applications with ease using comprehensive browser-base
 
 == Detail
 
-Business experts and application developers in organizations of any size need to be able to model, automate, measure, and improve their critical processes and policies. Red Hat(R) JBoss(R) BRMS makes it possible with fully integrated business rules management and complex event processing (CEP). All through an open source subscription.
+Learn how the Business Resource Planner can tackle a wide range of planning problems, from delivery routing to shift scheduling and
+even managing compute resources in the cloud. 
 
-image:#{cdn('images/products/brms/capture.png')}["BRMS Screenshot", width="518"]
+https://vimeo.com/album/2982444/video/143059283[image:#{cdn('images/products/brms/planner-video.png')}["Planner Video", width="518"]]
```
Comment (LightGuard) [inline products/brms/overview.adoc]: This should use the same code as what I suggested in RHD.

**Comment by eschabell** [comment]
Preview shows the suggested fix from RDS just dumping code to the page:

http://www-stg.jboss.org/pr/1002/build/2300/products/brms/overview/

Code duped half way down the page:

<a href="http://www-stg.jboss.org/pr/1002/build/2300/video/vimeo/143059283">Video Thumbnail</a><span class="label material-duration">03:12</span><h4><a href="http://www-stg.jboss.org/pr/1002/build/2300/video/vimeo/143059283">Solve optimization and scheduling problems with business resource planning</a></h4><p class="author">Author: </p><p class="material-datestamp">Added 2015-10-20</p><div class="body"><p>Learn how the Business Resource Planner can tackle a wide range of planning problems, from delivery routing to shift scheduling and even managing compu</p></div>

**Comment by LightGuard** [comment]
Hm, okay, looks like it isn't needed since you switched the intro video.

**Comment by paulrobinson** [comment]
@eschabell looks like a rebase is needed.

**Comment by eschabell** [comment]
Can someone rebase (mine does not work) and/or push this to production? It
is running on developer.redhat.com already.



>

**Comment by paulrobinson** [comment]
I've replaced this with: https://github.com/jboss-developer/www.jboss.org/pull/1009
